tests/integration/ceph_flex_test.go

```diff
@@ -220,6 +221,7 @@ func (s *CephFlexDriverSuite) TearDownSuite() {
 	s.testClient.BlockClient.DeleteStorageClass("rook-ceph-block-rwx")
 	s.testClient.PoolClient.DeletePool(s.testClient.BlockClient, s.namespace, "block-pool-rwo")
 	s.testClient.PoolClient.DeletePool(s.testClient.BlockClient, s.namespace, "block-pool-rwx")
+	time.Sleep(time.Second * 10)
```

Comment:
Again, can we query every few seconds for when the CR is actually removed? Otherwise the tests might still be flaky if it takes longer than this timeout.